### PR TITLE
yle-dl: 2.31 -> 20200807

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Motivation for this change

https://r.ryantm.com/log/2020-10-09.log

###### Things done

Updated to latest version and fixed tests when enabled.

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by sikmir** [comment]
@SuperSandro2000 Please, update license to `licenses.gpl3Plus`, since `licenses.gpl3` is deprecated (https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206/4).
See https://github.com/aajanki/yle-dl/blob/3100fa5f4f82583d092e580930b84cd3214860f1/setup.py#L56:
`        'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',`

**Comment by SuperSandro2000** [comment]
> @SuperSandro2000 Please, update license to `licenses.gpl3Plus`, since `licenses.gpl3` is deprecated ([discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206/4](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206/4)).
> See [aajanki/yle-dl@`3100fa5`/setup.py#L56](https://github.com/aajanki/yle-dl/blob/3100fa5f4f82583d092e580930b84cd3214860f1/setup.py#L56):
> ` 'License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)',`

@sikmir Changed that. Thanks for the notice.

**Comment by sikmir** [comment]
[20201022](https://github.com/aajanki/yle-dl/releases/tag/20201022) is out.

**Comment by SuperSandro2000** [comment]
> [20201022](https://github.com/aajanki/yle-dl/releases/tag/20201022) is out.

Updated. Thanks for the notice!

**Review by sikmir — CHANGES_REQUESTED** [review]

File: pkgs/tools/misc/yle-dl/default.nix [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
+  checkInputs = with python3Packages; [ ffmpeg_3 pytest pytestrunner ];
 
   meta = with stdenv.lib; {
     description = "Downloads videos from Yle (Finnish Broadcasting Company) servers";
     homepage = "https://aajanki.github.io/yle-dl/";
-    license = licenses.gpl3;
-    maintainers = [ maintainers.dezgeg ];
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ dezgeg SuperSandro2000 ];
     platforms = platforms.linux;
```
[the remark this answers is not in the document]
Comment (SuperSandro2000) [reply pkgs/tools/misc/yle-dl/default.nix]: Good catch. I already added the commit and we wait until php is fixed.

**Review by sikmir — APPROVED** [review]

**Comment by sikmir** [comment]
yle-dl moved to python3 https://github.com/NixOS/nixpkgs/issues/101964

**Comment by jonringer** [comment]
I was wondering why this looked so familiar https://github.com/NixOS/nixpkgs/pull/67656

**Review by jonringer — APPROVED** [review]
LGTM
shows usage

```
https://github.com/NixOS/nixpkgs/pull/100191
1 package built:
yle-dl
```

**Pull request merged**